documentation/airport-guides/kclt.md

```diff
@@ -0,0 +1,44 @@
+### About this facility
```

Comment:
The airport guide is quite short, some more information about the airport might be useful. You can also have a look at the other airport guides to get some inspiration.

Reply:
Honestly I don't have a lot of time to put into it. If someone else wants to do this, great, but I can't modify it.